test/examples_test.go

```diff
@@ -181,6 +181,18 @@ func getExamplePaths(t *testing.T, dir string) []string {
 			return filepath.SkipDir
 		}
 		if info.IsDir() == false && filepath.Ext(info.Name()) == ".yaml" {
+			// Ignore test matching the regexp in the TEST_EXAMPLES_IGNORES
+			// environement variable.
+			val, ok := os.LookupEnv("TEST_EXAMPLES_IGNORES")
+			if ok {
+				re := regexp.MustCompile(val)
+				submatch := re.FindSubmatch([]byte(path))
+				if submatch != nil {
+					t.Logf("Skipping test %s", path)
```

Comment:
Does t.Skip work here?

Reply:
@dlorenc it would skip all the tests I think :upside_down_face: